cache/models/samples.js

```diff
@@ -586,7 +596,7 @@ module.exports = {
    */
   bulkUpsertSample(sampleQueryBody) {
     const promises = sampleQueryBody.map(
-      (sampleReq) => upsertOneSample(sampleReq)
+      (sampleReq) => upsertOneSample(sampleReq, null, true)
     );
```

Comment:
Sorry. I can extract it from the sample name. I do not need it as a seperate field.